src/components/data-exploration/heatmap/HeatmapMetadataTrackSettings.jsx

```diff
@@ -47,8 +47,8 @@ const HeatmapMetadataTrackSettings = (props) => {
   const [trackData, setTrackData] = useState(getTrackData());
 
   const getUpdatedTrackData = () => _.unionBy(
-    trackData,
     getTrackData(),
+    trackData,
```

Comment:
According to the [_.unionBy docs](https://lodash.com/docs/4.17.15#unionBy), the updated value is taken from the first array, so `getTrackData()` should be put first because it contain the more updated data.